src/plugins/robots/drone/simulator/pointmass3d_drone_model.h

```diff
@@ -76,16 +77,21 @@ namespace argos {
       CVector3 m_cVelocityPrev;
       /* angular velocity of the drone */
       CVector3 m_cAngularVelocity;
-      CVector3 m_cAngularVelocityPrev;
+      CVector3 m_cPrevAngularVelocity;
       /* acceleration of the drone */
-      CVector3 m_cAccelerationPrev;
+      CVector3 m_cPrevAcceleration;
       /* angular acceleration of the drone */
-      CVector3 m_cAngularAccelerationPrev;
-      /* TODO: improve naming of the following variables */
```

Comment:
So this was a note to myself that was a bit unclear. The problem was that the naming of these variables wasn't particularly consistent with the other variables and how they are used in the equations.